index.html

```diff
@@ -713,7 +713,7 @@
 
 	<div class='slider-example'>
       <h3>Example 14:</h3>
-      <p>Using tick marks at specific positions..</p>
```

Comment:
these changes need to happen in the [index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl) file.

that file compiles down to this file